test/mix/tasks/phoenix_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Mix.Tasks.PhoenixTest do
+  use ExUnit.Case, async: true
+
+  test "prints version" do
+    Mix.Tasks.Phoenix.run []
+    assert_received {:mix_shell, :info, ["Phoenix v" <> _]}
+  end
+
+  test "provide a list of available phoenix mix tasks" do
+    Mix.Tasks.Phoenix.run []
+
+    assert_received {:mix_shell, :info, ["mix phoenix.digest " <> _]}
+    assert_received {:mix_shell, :info, ["mix phoenix.digest.clean " <> _]}
+    assert_received {:mix_shell, :info, ["mix phoenix.gen.channel" <> _]}
+    assert_received {:mix_shell, :info, ["mix phoenix.gen.html" <> _]}
+    assert_received {:mix_shell, :info, ["mix phoenix.gen.json" <> _]}
+    assert_received {:mix_shell, :info, ["mix phoenix.gen.model" <> _]}
```

Comment:
We don't need to assert on all of the tasks. Asserting for something like mix phoenix.gen.secret, digest and routes should be enough,